src/components/scenes/RequestScene.tsx

```diff
@@ -629,6 +629,6 @@ export const RequestScene = (props: OwnProps) => {
   const { pluginId = '' } = wallet.currencyInfo ?? {}
   const tokenId = getTokenId(account, pluginId, currencyCode)
 
-  const iconColor = useIconColor({ pluginId, tokenId: tokenId !== undefined ? tokenId : '' })
+  const iconColor = useIconColor({ pluginId, tokenId: tokenId !== undefined ? tokenId : null })
```

Comment:
This will load the icon color for the mainnet asset. Better not to load a color at all otherwise it will flicker between the two

Reply:
Why would it load the icon for the mainnet when there is a tokenId?